drivers/usb/device/usb_dc_sam_usbc_priv.h

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
You can just move everything that is here to usb_dc_sam_usbc.c, it is not that much.